package-examples/subpackages/wordpress/README.md

```diff
@@ -33,60 +33,50 @@ wordpress
           ├── [mysql-deployment.yaml]  PersistentVolumeClaim mysql-pv-claim
           ├── [mysql-deployment.yaml]  Deployment wordpress-mysql
           └── [mysql-deployment.yaml]  Service wordpress-mysql
-  
-  The fetched package contains [setters]. Invoke [list-setters] command to list
-  the [setters] recursively in all the packages.
-  
-  $ kpt cfg list-setters wordpress/
-  
-      wordpress/
-               NAME             VALUE      SET BY   DESCRIPTION   COUNT   REQUIRED  
-        gcloud.core.project   PROJECT_ID                          3       No        
-        image                 wordpress                           1       No        
-        tag                   4.8                                 1       No        
-        teamname              YOURTEAM                            3       Yes       
-      
-      wordpress/mysql/
-               NAME             VALUE      SET BY   DESCRIPTION   COUNT   REQUIRED  
-        gcloud.core.project   PROJECT_ID                          3       No        
-        image                 mysql                               1       No        
-        tag                   5.6                                 1       No        
-        teamname              YOURTEAM                            3       Yes       
-        
-   You may notice that the [auto-setter] `gcloud.core.project` is already set if you
-   have `gcloud` configured on your local.
-  
-  ### Provide the setter values
-  Provide the values for all the [required setters]. By default, [set] 
-  command is performed only on the resource files of provided package and not its 
-  subpackages. `--recurse-subpackages(-R)` can be leveraged to run the command on 
-  subpackages recursively.
-  
-  $ kpt cfg set wordpress/ teamname myteam -R
-  
-      wordpress/
-      set 3 field(s)
-      
-      wordpress/mysql/
-      set 3 field(s)
-  
-  ### Apply the package
-  
-  Apply all the contents of the package recursively to the cluster
-  
-  $ kubectl apply -f wordpress/ -R
+
+### Configure namespace (optional)
+
+By default, the packages will be deployed into `default` namespace. To provide a namespace, you must
+add [set-namespace] function to the pipeline definition in `wordpress/Kptfile`.
+
+    pipeline:
+      mutators:
+      - image: gcr.io/kpt-fn/set-namespace:VERSION
+        configMap:
+          namespace: <VALUE>
+
+### Configure setter values (optional)
+
+Setters are listed under `apply-setters` function in the pipeline definition of each package.
+You may declare new desired values for the setters by editing the `Kptfile` directly.
+
+### Other configurations (optional)
+
+You may choose to declare additional configurations from the list of [available functions].
```

Comment:
Not sure what this sentence is supposed to say, but it's vague enough that you probably want to delete it.

Reply:
Done.